prescription/prescription.go

```diff
@@ -0,0 +1,127 @@
+package prescription
+
+import (
+	"github.com/tidepool-org/platform/data/types/settings/pump"
+)
+
+const (
+	PrescriptionStateDraft     = "draft"
+	PrescriptionStatePending   = "pending"
+	PrescriptionStateSubmitted = "submitted"
+	PrescriptionStateReviewed  = "reviewed"
+	PrescriptionStateExpired   = "expired"
+	PrescriptionStateActive    = "active"
+	PrescriptionStateInactive  = "inactive"
+
+	PrescriptionTrainingInPerson = "inPerson"
+	PrescriptionTrainingInModule = "inModule"
+
+	PrescriptionTherapySettingInitial              = "initial"
+	PrescriptionTherapySettingTransferPumpSettings = "transferPumpSettings"
+	PrescriptionTherapySettingCPT                  = "cpt" // TODO: Certified Personal Trainer?
```

Comment:
https://en.wikipedia.org/wiki/Current_Procedural_Terminology

Reply:
Turned out to be Certified *Pump* Trainer